environment/lib/near2eth-relay.js

```diff
@@ -142,6 +142,7 @@ class Near2EthRelay {
     }
 
     async run () {
+        process.send('ready');
```

Comment:
what is this for?

Reply:
This signals to pm2 that the relayer has booted, done all initialization and is fully functional, see ` wait_ready: true,` configuration option.

Reply:
I see, does this need to be in the end of `run ()`?

Reply:
I should be between initialization and the main loop of the relay. If we put it at the end of run then it will never be emitted and pm2 will think this service is not "ready".